lib/chartmogul/csv/invoice.rb

```diff
@@ -3,15 +3,14 @@
 module ChartMogul
   module CSV
     class Invoice < Base
-      INVOICE_HEADERS = %w[Customer\ external\ ID Invoice\ external\ ID Invoiced\ date Due\ date Currency Status].freeze
+      INVOICE_HEADERS = %w[Customer\ external\ ID Invoice\ external\ ID Invoiced\ date Due\ date Currency].freeze
```

Comment:
Why? I thought we are gonna store the "voided" status here, or am I missing something?

Reply:
well, it is a question of order. We will be deploying charges first and then support of open invoices.